pkg/image/api/helper.go

```diff
@@ -944,6 +948,20 @@ func PrioritizeTags(tags []string) {
 				minor = append(minor, v)
 				continue
 			}
+		case reMajorWithPatch.MatchString(short):
```

Comment:
Not quite, they're stating to be compliant with [semver.org](http://semver.org/) which specifically says `A normal version number MUST take the form X.Y.Z`. The problem is we will deal with all sorts of version schemes, see for example the enclosed bug which uses `9.4-6, 9.4-6.1, 9.4-6.2, 9.4-6.4` which won't work not even with this patch :/ But it's almost impossible to support all possible versioning scheme. "Fixing" those missing Y or Z part is doable and we already are doing that. What I'm more worried here is that additionally we'll sort `9 < 9.4 < 9.4.1` due to the split, which worries me the most.